test/filters/test_motion.py

```diff
@@ -0,0 +1,66 @@
+from typing import Tuple
+import pytest
+
+import kornia
+import kornia.testing as utils  # test utils
+from test.common import device_type
+
+import torch
+from torch.autograd import gradcheck
+from torch.testing import assert_allclose
+
+
+@pytest.mark.parametrize("ksize", [3, 11])
+@pytest.mark.parametrize("angle", [0., 360.])
+@pytest.mark.parametrize("direction", [-1., 1.])
+def test_get_motion_kernel2d(ksize, angle, direction):
+    kernel = kornia.get_motion_kernel2d((ksize, ksize), angle, direction)
+    assert kernel.shape == (ksize, ksize)
+    assert kernel.sum().item() == pytest.approx(1.0)
+
+
+class TestMotionBlur:
+    @pytest.mark.parametrize("batch_shape", [(1, 4, 8, 15), (2, 3, 11, 7)])
+    @pytest.mark.parametrize("device_type", ("cpu", "cuda"))
+    def test_motion_blur(self, batch_shape, device_type):
```

Comment:
don't use `device_type` yet, this is coming in a new PR https://github.com/kornia/kornia/pull/313

Reply:
Fixed